README.md

```diff
@@ -41,12 +41,6 @@ The timestamp is always given in millisecond but the time resolution can be redu
 The use case is to be able to compare a build time to a file time attribute.
 On node v0.10 or with file system like HFS or FAT, `fs.stat` time attributes like `mtime` precision is one second.
```

Comment:
@sttk Can we change this line to:
```suggestion
On some file systems, `fs.stat` time attributes like `mtime` might have one second precision.
```

We won't support node 0.10 anymore and I'm not sure if those filesystems still have 1 second precision with new nodejs.

Reply:
It looks like I can just commit this.